### DISCUSSION TITLE
Request for trying new Blob storage

### DISCUSSION THREAD

**Discussion by renatodex** [vercel/community#3670] [Feature Question]
### Question

Hey, I'm not sure how dumb it is to post it here, but I'm working in an application on Vercel, and would like to understand how hard would be to receive an invite to test Blog storage system.

Currently we are hosting images on S3, and it would be AWESOME to see how well Vercel handle this with the new Blob storage system.

Thank you !

**Comment by amyegan** [comment]
Hey @renatodex. If you [submit a case](https://vercel.com/help#issues) to the Customer Success Team, we might be able to get you early access to test Vercel Blob 😉
